src/date-input/template.test.js

```diff
@@ -205,7 +205,6 @@ describe('Date input', () => {
   it('passes through fieldset params without breaking', () => {
     const $ = render('date-input', examples['default'])
 
-    expect(htmlWithClassName($, '.govuk-c-error-message')).toMatchSnapshot()
```

Comment:
Should this have moved or been split out into a different test rather than just being removed?

Reply:
Hmm, yeah, I'll do a PR.

Reply:
See https://github.com/alphagov/govuk-frontend/pull/625

Reply:
sorry

Reply:
S'all good, slipped my mind too 👍